pocs/utils/images/__init__.py

```diff
@@ -103,14 +104,22 @@ def _make_pretty_from_fits(fname, **kwargs):
         ax = plt.subplot(projection=wcs)
 
         ax.coords.grid(True, color='white', ls='solid')
-        ax.set_xlabel('RA')
-        ax.set_ylabel('Dec')
+
+        ra = ax.coords[0]
+        dec = ax.coords[1]
+
+        ra.set_axislabel('Right Ascension')
```

Comment:
I tend to do `label [units]` (although I think @AnthonyHorton does something different), in which case this would be `Right Ascension [hms]` and `Declination [dms]`.

Reply:
Yes, I tend to prefer the `label / units` convention (i.e. the numbers are the quantity in question divided by the given unit).  Either is fine, though.

In this particular case putting units in the axis labels should be redundant though, the tick label formats include units, see https://github.com/AstroHuntsman/HuntsmanPOCSLegacy/blob/develop/examples/notebooks/dice9.png for an example.

Reply:
I have written them in for now. Will make sure that the units are in the axis label when I run the code with a fits file that has wcs., then I will get rid of the units in the label.